townsfolk/investigative/huntsman

```diff
@@ -10,7 +10,11 @@ Each night, the Huntsman may check a target to determine if they are part of the
 They are not affected by any disguises, but are affected by redirections.
 
 __Formalized__
-Immediate Night: Attribute Investigate @Selection for `GroupMembership:Wolfpack`
+Immediate Night: |investigating.role.1|
+  • Process: Attribute Investigate @Selection for `GroupMembership:Wolfpack`
+  • Evaluate:
+    ‣ @Result is True: `@Selection is a member of the Wolfpack`
```

Comment:
Do we capitalize Wolfpack here?

Reply:
I have lower cased it.

Reply:
https://github.com/WerewolvesRevamped/Werewolves-Roles/pull/1542/commits/8e02b36a67b5c7b33a9c478692708552807768c9